### PR TITLE
Nav bar improvements

### PR DESCRIPTION
**Pull request by drewbern**
Adds nav bar with buttons. Adds category drop-down for Country pages. Cut and paste design from w3school.com, so we should definitely go back in and change the design/colors to our liking.

### CODE REVIEW

**Comment by drewbern** [comment]
Addresses #77

File: src/main/java/com/google/codeu/servlets/CountryServlet.java [hunk 1]
```diff
@@ -86,12 +86,12 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
     if (currentUser == null) request.setAttribute("currentUser", null);
     else request.setAttribute("currentUser", currentUser.getEmail());
 
-    request.setAttribute("code", countryCode);
+    request.setAttribute("countryCode", countryCode);
```
Comment (nbaldy) [inline src/main/java/com/google/codeu/servlets/CountryServlet.java:89]: Thanks for the switch! I agree that's a better name

File: src/main/webapp/WEB-INF/category.jsp [hunk 2]
```diff
@@ -20,11 +20,14 @@
...
-      </ul>
-    </nav>
+    <div class="navbar">
+      <a href="/">Home</a>
+    <% if (currentUser != null) { %>
+      <a href="/users/<%=currentUser%>">Your Page</a>
+    <% } else { %>
+      <a href="/login">Login</a>
+    <% } %>
+    </div>
```
Comment (nbaldy) [inline src/main/webapp/WEB-INF/category.jsp:30]: Good call on this stuff. I'd been meaning to file  a bug but never got around to it

**Review by nbaldy — APPROVED** [review]
Looks good to me!

**Comment by nbaldy** [comment]
I am going to file an enhancement issue for making the style match everything else, but I think this definitely works to get the functionality down

**Comment by drewbern** [comment]
Addresses #86

**Pull request merged**